### PR TITLE
docs(content switcher): add icon-only variant

### PR DESCRIPTION
**Pull request by aagonzales**
Closes https://github.com/carbon-design-system/carbon-website/issues/3504

Added docs for icon content switcher variant
- updated usage tab
- updated style tab

### CODE REVIEW

**Review by tay1orjones — APPROVED** [review]
Looks awesome!

**Review by alisonjoseph — APPROVED** [review]
LGTM!

File: src/pages/components/content-switcher/usage.mdx [hunk 1]
```diff
@@ -105,38 +115,51 @@ binary decision.
...
+![Anatomy of a content switcher.](images/contentswitcher-usage-2.png)
 
 </Column>
 </Row>
 
 1. **Selected content tab:** Only one content tab can be selected at a time and
    there should always be one selected.
 2. **Content tab:** Selectable container for each content view.
-3. **Text label:** Describes the content view.
+3. **Label:** Text label or iccon that describes the content view.
```
Comment (laurenmrice) [inline src/pages/components/content-switcher/usage.mdx:126]:
```suggestion
3. **Label:** Text label or icon that describes the content view.
```

**Review by laurenmrice — CHANGES_REQUESTED** [review]
Looks awesome! Just a couple typo comments:

**Review by laurenmrice — APPROVED** [review]

**Pull request merged**